src/Moryx.Products.UI.Interaction/Recipes/Details/Default/DefaultRecipeDetailsViewModel.cs

```diff
@@ -8,6 +8,6 @@ namespace Moryx.Products.UI.Interaction
     [RecipeDetailsRegistration(DetailsConstants.DefaultType)]
     internal class DefaultRecipeDetailsViewModel : RecipeDetailsViewModelBase
     {
-
+        public bool IsClassificationEditable { get => IsEditMode && EditableObject.IsClone ; }
```

Comment:
I am pretty sure the condition is inverted. Shouldn't `IsClone` do the opposite?

Reply:
This could be done fully in UI by DataTrigger and should not be part of ViewModel